examples/js/effects/VREffect.js

```diff
@@ -42,8 +42,26 @@ THREE.VREffect = function ( renderer, onError ) {
 
 			if ( onError ) onError( 'HMD not available' );
 
+			return;
```

Comment:
yep, because I want to initialise [the legacy API/Fullscreen logic](https://github.com/mrdoob/three.js/pull/8330/files#diff-9241b49ed6cabf364c42e019568a978eR51) only if there's a `vrHMD`.

it'd be more verbose, but I could probably make it more readable by explicitly checking for `vrHMD` below.